docs/slis/argocd.md

```diff
@@ -0,0 +1,111 @@
+
+
+## Goal
+
+Identify key SLIs for ArgoCD and document them in the [ArgoCD Runbook](https://github.com/AICoE/argocd-customization). These SLIs should be used to define our SLOs. The aim with SLOs is to have as few SLOs as possible.
+
+**SLIs**
+
+SLI stands for Service Level Indicator -- more information can be found in the [Site Reliability Engineering](https://landing.google.com/sre/sre-book/chapters/service-level-objectives/) book.  
+
+**Measuring Correctness**
+
+ArgoCD also has support for health checks in applications. I think this isn’t necessary for us to use right now from a SLI perspective ( since this is for individual applications), but it could be useful in the future for yet another check to monitor our applications
+
+
+<table>
```

Comment:
I guess this is because the google doc -> markdown convert did that. 
I would use markdown tables https://help.github.com/en/github/writing-on-github/organizing-information-with-tables#creating-a-table

Reply:
oh man but converting that to markdown tables is going to be such a pain :cry: